express-notes-api/index.js

```diff
@@ -0,0 +1,147 @@
+const express = require('express');
+const app = express();
+
+const dataObject = require('./data.json');
+const fs = require('fs');
+
+app.get('/api/notes', (req, res) => {
+  fs.readFile('./data.json', 'utf8', err => {
+    if (err) {
```

Comment:
You can remove all of these `fs.readFile` calls now that you are requiring `data.json`